docs(purpose): update copy around design systems (#1072)
ionic-team/stencil-site: docs/reference/faq.md

### Who uses Stencil?

> Stencil was initially developed for Ionic Framework and has been a very successful Web Component-based library & UI framework. Web Components are now in thousands of app store apps, and nearly 4 million new Ionic Framework projects are being created every year.

Reviewer: should this read 'which is a very successful...' ? because it sounds like it's talking about Ionic Framework (edit applies before what you are changing here)

```suggestion
Stencil was initially developed for Ionic Framework which is a very successful Web Component-based library & UI framework. Web Components are now in thousands of app store apps, and nearly 4 million new Ionic Framework projects are being created every year.
```

Author: 👍 Updated all the v3 versions, as well as 'next' in https://github.com/ionic-team/stencil-site/pull/1072/commits/d9ea1643227c61e9a29f38b7d97c411672a24ebf
